### PR TITLE
Add 2.1.0.rc1 announcement post

### LINKED ISSUE #579: Write 2.1.0.rc1 release blog post

### PR DESCRIPTION
**Pull request by timriley**
Ready for you to review, @jodosha :)

When it comes to publishing this, **don't forget to update the `date:` in the front matter!** 😜

---

Closes https://github.com/hanami/hanami.github.io/issues/579

### CODE REVIEW

File: source/blog/2023-10-24-hanami-210rc1.html.markdown [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+By default, this file auto-installs the [Foreman](https://github.com/ddollar/foreman) gem, so you can run your Hanami dev servers without any other setup.
+
+This file is yours to own and modify, so you can feel free to add any other dev server affordances here.
+
+## Parts and helper adjustments
+
+After beta2, we made some refinments to our assets helper names to minimise naming collisions in your views. The adjusted names are `javascript_tag`, `stylesheet_tag`, `favicon_tag`, `image_tag`, `video_tag` and `audio_tag`.
+
+In addition, helpers inside your view part classes are now contained inside a single `helpers` object, ensuring no collisions with the many method names that view parts forwrd to your app’s domain objects. Accessing a helper inside a view part new looks like this:
```
Comment (waiting-for-dev) [inline source/blog/2023-10-24-hanami-210rc1.html.markdown]:
```suggestion
In addition, helpers inside your view part classes are now contained inside a single `helpers` object, ensuring no collisions with the many method names that view parts forward to your app’s domain objects. Accessing a helper inside a view part new looks like this:
```

**Review by waiting-for-dev — APPROVED** [review]
Awesome content! I only found a typo :slightly_smiling_face:

File: source/blog/2023-10-24-hanami-210rc1.html.markdown (hunk 1 above)
Comment (timriley) [reply source/blog/2023-10-24-hanami-210rc1.html.markdown]: Thanks! Pushed a fix for this.

**Comment by timriley** [comment]
n.b. at this point we'll need to change both the file name _and_ the date before posting (since we delayed this by a week).

In the meantime, this let me look at the deploy preview today, and it seems like the images in the post are 404ing. Is there a different place I need to put them, @jodosha?

**Comment by timriley** [comment]
Have just pushed updates to this anticipating the new release date: 2023-11-01

**Review by jodosha — APPROVED** [review]
@timriley Ready to go! 🏁

**Pull request merged**